test/server_eviction.py

```
@@ -226,67 +286,94 @@ def test_weight_factor_protects_model(self):
+                        timeout=RACE_REQUEST_TIMEOUT,
                     )
             except Exception as exc:  # noqa: BLE001 - surfaced via errors list
                 errors.append(exc)
+                stop.set()
 
-        threads = [threading.Thread(target=churn) for _ in range(3)]
-        for t in threads:
-            t.start()
-        # Run the churn long enough to span several eviction-engine cycles (5s each).
-        time.sleep(20)
+        def drive_idle_evaluation():
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `drive_idle_evaluation`

Add return type annotation: `None`

(ANN202)